config/lib.json

```diff
@@ -400,6 +420,21 @@
             "zlib"
         ]
     },
+    "libjxl": {
+        "source": "libjxl",
+        "static-libs-unix": [
+            "libjxl.a",
+            "libjxl_dec.a",
+            "libjxl_threads.a",
+            "libhwy.a"
```

Comment:
this is not part of libjxl, but libjxl depends on libhwy, we add it here because it's built with libjxl and because we need to specify it for php make EXTRA_LIBS